##############################################################################
# About
##############################################################################

# Unused yet, but basic idea is to use this to feed gopher behaviours with the correct names/topics for your
# robot so that you don't have to 1) hardcode everything, or 2) do massive amounts of params/remaps in launchers.

##############################################################################
# Names
##############################################################################
topics:
  display_notification: /gopher/status_notifier/display_notification
  initial_pose: /navi/initial_pose
  switch_map: /dslam/switch_map

namespaces:
  semantics: /rocon/semantics

##############################################################################
# Aliases
##############################################################################

# Button aliases standardised for gopher deliveries,
buttons:
  go:   /gopher/buttons/go
  stop: /gopher/buttons/stop

# Sound aliases standardised for gopher deliveries,
# system wide aliases should be specified above. 
sounds:
  honk: /gopher/commands/sounds/groot

